### PR TITLE
modify comment

### PR DESCRIPTION
**Pull request by shilpapullayikodi**

### CODE REVIEW

File: html-and-the-web_div-salad/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  </head>
+  <!--someone forgot to wrap the content of our webpage with an HTML element,
+	which one was it again?-->
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
+      <!--Which elements could be used to link to parts of the webpage,
+		which appear below?-->
+      <nav id="nav">
+        <a href="about-me">About Me</a>
```
Comment (KristofferJans) [inline html-and-the-web_div-salad/index.html:22]: add # before id

File: html-and-the-web_div-salad/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <a href="contact">Contact</a>
```
Comment (KristofferJans) [inline html-and-the-web_div-salad/index.html:23]: same #

File: html-and-the-web_div-salad/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        <a href="skillset">Skillset</a>
```
Comment (KristofferJans) [inline html-and-the-web_div-salad/index.html:24]: #

File: html-and-the-web_div-salad/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      </nav>
+    </header>
+    <!--Looks like the following part is the central content of our webpage.-->
+    <main>
+      <section id="about-me">
+        <h2>About me</h2>
+        <article>
```
Comment (KristofferJans) [inline html-and-the-web_div-salad/index.html:31]: just put p for paragraph when theres only text

File: html-and-the-web_div-salad/index.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        <section>
+          <img
+            src="https://source.unsplash.com/random/400x300?fruit"
+            alt="a random picture"
+          />
+          <figcaption>
+            A random picture, because I would like to maintain my privacy and
+            let my code speak for me, instead of images.
+          </figcaption>
+        </section>
```
Comment (KristofferJans) [inline html-and-the-web_div-salad/index.html:46]: idk why there is 2x section?

File: html-and-the-web_div-salad/index.html (hunk 1 above)
Comment (shilpapullayikodi) [reply html-and-the-web_div-salad/index.html]: Added 👍

File: html-and-the-web_div-salad/index.html (hunk 2 above)
Comment (shilpapullayikodi) [reply html-and-the-web_div-salad/index.html]: Added 👍

File: html-and-the-web_div-salad/index.html (hunk 5 above)
Comment (shilpapullayikodi) [reply html-and-the-web_div-salad/index.html]: Changed to figure instead of section

**Pull request merged**